src/main/resources/migrations/00001_add_catalogues_table.sql

```diff
@@ -12,3 +12,5 @@ CREATE TABLE catalogues (
 );
 --rollback drop table catalogues;
 
+--changeset uk.gov.pay:add_catalogues_external-service-id_index
+CREATE INDEX external_service_id_idx ON catalogues (external_service_id);
```

Comment:
I don't think this works - liquibase calculates checksums of already executed changes and stores them _somewhere_, so if you modify a changeset that's already been executed it'll complain and fail. I think you need to add a new changeset (`00003_blah`) with these changes. Worth running by Kasun, Raul or someone.